main.scss

```diff
@@ -23,7 +23,10 @@
 @import 'scss/wrapped_general';
 
 @if $o-typography-is-silent == false {
-	@include oFontsIncludeAll();
+
+	@if $o-typography-load-fonts == true {
```

Comment:
Ok, fine. It's probably a grey line regarding the spec, but you're right, better like this